test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetAudioStreamingIndicator] SDL must transfer request from mobile app to HMI in case no any failures
+-- [GeneralResultCodes] INVALID_DATA wrong type
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- In case media app sends the invalid SetAudioStreamingIndicator_request to SDL
+-- and this request is allowed by Policies
+-- SDL must NOT transfer SetAudioStreamingIndicator_request to HMI
+-- SDL must respond with result code INVALID_DATA
+--
+-- 1. Used preconditions
+-- Allow SetAudioStreamingIndicator RPC by policy
+-- Register and activate media application
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = nil),
```

Comment:
@istoimenova, according to ideas file script should check empty_value for audioStreamingIndicator parameter
As far as I understand empty means blank, i.e. "", and nil means absence of parameter
We check absence of parameter in 025 script